### ISSUE TITLE
A11y_AzureToolsStorageExplorer_UploadFolder_AI4D: The name of the check box is not provided for the Upload .vhd/vhdx files as page blobs (recommended)

### ISSUE THREAD

**Issue by Radha2019** [microsoft/AzureStorageExplorer#1883]
[“Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS4.1.2; #AI4D; #Win10;

Environment Details:
Windows Version: Windows10

**Steps to Reproduce:**

1. Launch Storage Explorer
2. Navigate to "Add Account" menu item present on left side of screen and hit enter.
3. Select "Add an Azure Account" radio button.
4. And Select "Azure" from "Azure Environment" combo box and click on Next button.
5. Sign in to Azure.
6. After having connected to Azure, go to "Manage Accounts" pane and hit enter.
7. Select the subscriptions you will use. Once selected, click on "Apply" button.
8. Toggle Explorer pane gets opened in left.
9. Navigate to a Storage account and expand it.
10. Navigate to "Blob Containers" and right click on it.
11. Select "Create Blob container" and Create a new Blob container.
12. Once completed, navigate to the right side of Storage Explorer.
13. Navigate to "Upload" button present on toolbar and click on it.
14. Select "Upload Files" and hit enter.
15. "Upload Files" Popup gets opened.
16. Navigate to check box provided under Blob type and Open AI4D 
17. check whether the name is provided for the check box .

**Actual:**
The name of the check box is not provided for the Upload .vhd/vhdx files as page blobs (recommended)
**Expected:** 
The name of the check box should be  provided for the Upload .vhd/vhdx files as page blobs (recommended) as written besides to check box.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

User Impact: Screenreader user unable to know the name of the check box.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
**Attachment For Reference:**
![1883_ Accessibility A11y_AzureToolsStorageExplorer_UploadFolder_AI4D](https://user-images.githubusercontent.com/55691739/65425546-ddf08c00-de2b-11e9-819a-2da12dd9d236.png)

**Comment by Radha2019** [comment]
[https://accessibilityinsights.io/](url)

**Comment by MRayermannMSFT** [comment]
Jason merged a fix for this into master.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is no longer to reproduces, Hence closing this bug.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
